# request
float64[] positions
float64 speed
float64 timeout
---
# response
bool done
